pkgs/applications/networking/cluster/k9s/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "kubernetes-k9s-${version}";
```

Comment:
```suggestion
  pname = "kubernetes-k9s";
```

Reply:
Could you explain what is the difference?

Reply:
As per the [19.03 release notes][1]:
> `stdenv.mkDerivation` may now be passed `pname` and `version` attributes instead of `name`, and will now automatically construct the package name by joining them with a dash.

[1]: https://nixos.org/nixos/manual/release-notes.html#sec-release-19.03-notable-changes